src/download/download_wrapper.h

```diff
@@ -74,7 +74,7 @@ class DownloadWrapper {
   void                finished_download();
 
   std::unique_ptr<DownloadMain> m_main;
-  std::unique_ptr<Object>       m_bencode;
+  Object*                       m_bencode;
```

Comment:
It seems you're reverting changes.

Reply:
This was not done by me: https://github.com/rakshasa/libtorrent/commit/53f7062513676

Reply:
You're reverting the change change in that ref afaics.